libraries/tools/kotlin-gradle-plugin/src/main/kotlin/org/jetbrains/kotlin/gradle/report/data/BuildOperationRecord.kt

```diff
@@ -5,20 +5,17 @@
 
 package org.jetbrains.kotlin.gradle.report.data
 
-import org.gradle.api.Task
-import org.gradle.api.tasks.TaskState
 import org.jetbrains.kotlin.build.report.metrics.BuildMetrics
 
-interface TaskExecutionData {
-    val taskPath: String
-    val startMs: Long
-    val endMs: Long
+/** Data for a build operation (e.g., task or transform). */
+interface BuildOperationRecord {
+    val path: String
+    val classFqName: String
+    val isKotlin: Boolean
```

Comment:
Nit: imho `isKotlin` is not very self-describing. Probably better would be `isFromKotlinPlugin`

Reply:
That sounds good - Done.